status/statusgo_backend_new/saved_addresses.nim

```diff
@@ -0,0 +1,16 @@
+import json
+import ./core, ./response_type
+
+export response_type
+
+proc addSavedAddress*(name, address: string) =
+  let payload = %* [{"name": name, "address": address}]
+  discard callPrivateRPC("wallet_addSavedAddress", payload)
+
+proc deleteSavedAddress*(address: string) =
+  let payload = %* [address]
+  discard callPrivateRPC("wallet_deleteSavedAddress", payload)
```

Comment:
I think the design of the status-lib is that they always return `RpcResponse[JsonNode] {.raises: [Exception].}`
then the service can decide to discard that or not.